packages/blocks/src/_common/components/block-caption.ts

```diff
@@ -138,17 +138,26 @@ export class BlockCaptionEditor<
     ></textarea>`;
   }
 
-  @property({ attribute: false })
-  accessor block!: BlockComponent<Model> & { isInSurface?: boolean };
-
   @state()
   accessor caption: string | null | undefined = undefined;
 
   @state()
   accessor display = false;
 
+  @property({ attribute: false })
```

Comment:
We have the context here. You can use that.
https://github.com/toeverything/blocksuite/blob/master/packages/framework/block-std/src/view/element/lit-host.ts#L36